File: Dart/Hexadecimal to Decimal.dart
void main() {
  // Input a hexadecimal number
  print("Enter a hexadecimal number: ");
  String hexNumber = stdin.readLineSync()!;

  // Convert hexadecimal to decimal
  try {
    int decimalNumber = int.parse(hexNumber, radix: 16);
    print("The decimal equivalent of $hexNumber is: $decimalNumber");
  } catch (e) {
    print("Invalid hexadecimal input.");
  }
}

File: Python/Hexadecimal_to_Decimal.py
hexadecimal_number = input("Enter a hexadecimal number: ")

try:
    decimal_number = int(hexadecimal_number, 16)
    print(f"The decimal equivalent of {hexadecimal_number} is {decimal_number}")
except ValueError:
    print("Invalid input. Please enter a valid hexadecimal number.")
